Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml.cs

```diff
@@ -125,6 +127,12 @@ public POMEditPage(ApplicationPOMModel POM, eRIPageViewMode editMode = eRIPageVi
 
             mPomAllElementsPage = new PomAllElementsPage(mPOM, PomAllElementsPage.eAllElementsPageContext.POMEditPage);
             xUIElementsFrame.Content = mPomAllElementsPage;
+            //filre POM Activities to show
+            ObservableList<Activity> sharedActivities = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<Activity>();
+            ObservableList<Activity> pomActivities = GingerCore.General.ConvertListToObservableList(sharedActivities.Where(x => x.Guid == POM.ActivitiesGuid.FirstOrDefault(y => y == x.Guid)).ToList());
```

Comment:
change sharedActivities.Where(x => x.Guid == POM.ActivitiesGuid.FirstOrDefault(y => y == x.Guid)).ToList())
to
sharedActivities.Where(x =>  POM.ActivitiesGuid.Contains(x.Guid))